src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -470,7 +471,7 @@ void createConsumerWithNotExistingParameter(VertxTestContext context) throws Int
 
     @Test
     void receiveSimpleMessage(VertxTestContext context) throws InterruptedException, ExecutionException, TimeoutException {
-        String topic = "receiveSimpleMessage";
+        String topic = "receiveSimpleMessage-1";
```

Comment:
why adding this "-1" to topic names (I see it's done for others) fixes problems?

Reply:
It should work without these suffixes but AFAIK when I deploy external Kafka and external Bridge it fixes problems otherwise I got an error with a topic already exists.

Reply:
So it means that there is another problem somewhere, maybe not deleting topics at the end of the tests?

Reply:
Could be the issue, that `AdminClient` does not delete them properly.